docs/tutorials/new-component/types.ts

```diff
@@ -1,12 +1,10 @@
 /* Hier kommen alle Typen rein, die von der Komponente verwendet werden; 'States' und 'Props' werden in der Komponente mindestens benötigt. */
 
 import type { Generic } from 'adopted-style-sheets';
-import { Events } from '../../enums/events';
-import { EventValueOrEventCallback } from '../../types/callbacks';
-import { HeadingLevel } from '../../types/heading-level';
+import type { HeadingLevel, EventValueOrEventCallback } from '@public-ui/schema';
 
 export type KoliBriNewComponentCallbacks = {
-	[Events.onClick]?: EventValueOrEventCallback<Event, boolean>;
+	onClick?: EventValueOrEventCallback<Event, boolean>;
```

Comment:
I am not sure if there is another method to import the onClick event

Reply:
Ne, das muss so bleiben, weil das quasi als Property-Name als String angegeben wird.

Reply:
Der enum war aus dem Schema nicht exportiert. Ist nun exportiert. War irritiert, dass dieser noch nicht exportiert war.

Reply:
Danke für deinen PR. Er korrigiert einiges in der Doku. Die Enums sind in TS eher nicht zu empfehlen. Wir verwenden es effektiv fast nur bei dem Types.

Enum sollte lieber nicht exportiert werden, da sie sonst außerhalb verwendet werden. Sie sollten im SCHEMA bleiben.

Die types zu den Components gehören jüngs auch ins Schema, so braucht es auch nicht exportiert werden.